var/spack/repos/builtin/packages/llvm-openmp/package.py

```diff
@@ -18,10 +18,20 @@ class LlvmOpenmp(CMakePackage):
 
     depends_on('cmake@2.8:', type='build')
 
+    variant('multicompat', default=False,
+            description="Support gomp as well as the Intel openMP runtime library.")
```

Comment:
Flake is complaining about this line, can you split it up?

Reply:
Or put it on a single line

Reply:
@adamjstewart @scheibelp - Hi - I fixed the flake8 issue and pushed the change to this branch.  Sorry.   I didn't have flake8 on the system I was on and thought the code was ok.   But it wasn't.   Thanks,